data/assets/scene/digitaluniverse/milkyway_arm_labels.asset

```diff
@@ -39,7 +39,7 @@ local Object = {
     },
   },
   GUI = {
-    Name = "Milky Way Arms Labels",
+    Name = "Milky Way Arm Labels",
```

Comment:
```suggestion
    Name = "Milky Way Arms Labels",
```

The scene graph node does show multiple arms

Reply:
Yes, but we don't say pulsars labels, we say pulsar labels. A small grammatical detail, but I'm kind of a stickler for those. I didn't catch it in the last update.